packages/beacon-node/src/util/graffiti.ts

```diff
@@ -8,6 +8,13 @@ export function toGraffitiBuffer(graffiti: string): Buffer {
   return Buffer.concat([Buffer.from(graffiti, "utf8"), Buffer.alloc(GRAFFITI_SIZE, 0)], GRAFFITI_SIZE);
 }
 
+/**
+ * Converts a graffiti from 32 bytes buffer back to a UTF-8 string
+ */
+export function fromGraffitiBuffer(graffiti: Uint8Array): string {
+  return Buffer.from(graffiti).toString("utf8");
```

Comment:
it's worth to apply this pattern for future usage https://github.com/ChainSafe/lodestar/pull/6723/files#diff-3f51b963175b2a5dfd7853dd533277b5af873dc666944bcbed528a8a51d370c0R209

Reply:
ah good call, should definitely use no copy approach